wzdx/standard_to_enhanced/planned_events_translator.py

```diff
@@ -1,53 +1,60 @@
-
 import argparse
 import json
 import logging
 import copy
 import uuid
 
-from wzdx.sample_files.validation_schema import wzdx_v40_feed, road_restriction_v40_feed
-
-from wzdx.tools import date_tools, wzdx_translator
+from ..sample_files.validation_schema import (
+    work_zone_feed_v42,
+    road_restriction_v40_feed,
+)
 
-PROGRAM_NAME = 'PlannedEventsTranslator'
-PROGRAM_VERSION = '1.0'
+from ..tools import date_tools, wzdx_translator, uuid_tools
 
-
-DEFAULT_PLANNED_EVENTS_FEED_INFO_ID = '49253be7-0c6a-4a65-8113-450f9041f989'
+PROGRAM_NAME = "PlannedEventsTranslator"
+PROGRAM_VERSION = "1.0"
 
 
 def main():
     input_file, output_file = parse_planned_events_arguments()
 
-    planned_events_obj = json.loads(open(input_file, 'r').read())
+    planned_events_obj = json.loads(open(input_file, "r").read())
     wzdx = wzdx_creator(planned_events_obj)
     try:
-        event_type = wzdx['features'][0]['properties']['core_details']['event_type']
+        event_type = wzdx["features"][0]["properties"]["core_details"]["event_type"]
     except:
-        event_type = 'work-zone'
+        event_type = "work-zone"
     schemas = {
-        'work-zone': wzdx_v40_feed.wzdx_v40_schema_string,
-        'restriction': road_restriction_v40_feed.road_restriction_v40_schema_string
+        "work-zone": work_zone_feed_v42.wzdx_v42_schema_string,
+        "restriction": road_restriction_v40_feed.road_restriction_v40_schema_string,
     }
 
     if not wzdx_translator.validate_wzdx(wzdx, schemas[event_type]):
         logging.error(
-            'validation error more message are printed above. output file is not created because the message failed validation.')
+            "validation error more message are printed above. output file is not created because the message failed validation."
+        )
         return
-    with open(output_file, 'w') as fwzdx:
-        fwzdx.write(json.dumps(wzdx, indent=2))
-        print('Your wzdx message was successfully generated and is located here: ' + str(output_file))
+    with open(output_file, "w") as fWzdx:
+        fWzdx.write(json.dumps(wzdx, indent=2))
+        print(
```

Comment:
Print to logging swap

Reply:
Updated other logging statements to use print